lib/vector-config-macros/src/configurable.rs

```diff
@@ -1,7 +1,7 @@
 use proc_macro::TokenStream;
 use proc_macro2::Span;
-use quote::quote;
-use syn::{parse_macro_input, parse_quote, DeriveInput, ExprPath, Ident, Type};
+use quote::{quote, quote_spanned};
```

Comment:
Any of the changes in this file that have the term "span" or "spanned" in them are related to the "better code underlining in errors" stuff.